src/contract_cleaning.py

```diff
@@ -1,19 +0,0 @@
-def get_one_hot_encoded_contract(contract):
```

Comment:
Removed file contract_cleaning and added all functions into cleaning_functions since we used them in more than one feature